algopy_testing/src/algopy/primitives/bytes.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import annotations
 
 import base64
+import operator
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     from collections.abc import Iterator
 
+from itertools import zip_longest
+
 from algopy._constants import MAX_BYTES_SIZE, MAX_UINT64
```

Comment:
@daniel-makerx should we rename it to constants from _constants if its meant to be accessed from sibling namespaces like tests? I could also imagine this being useful in testing scenarios, constants like max byte size and etc would be handy to be accessible via the package instead of having to redefine them

Reply:
Can rename this to constants, however we will need a different approach to expose to consumers as the stubs definitions won't contain these declarations.

I'm thinking we will need something like an `algopy_testing` module that is provided in the wheel alongside these stubs implementations to provide additional functionality that isn't declared in the stubs. Such as utility functions, the testing context and things like these constants.

Reply:
Cool. I will remove the reference to `algo-y._constants` and re-implement them in tests just like in uInt64 test for now.